pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc

```diff
@@ -0,0 +1,24 @@
+[id='about-pantheon_{context}']
+= About Pantheon
+
+Pantheon is a content-management system and publishing platform developed by Red Hat. You can use it to publish documents based on AsciiDoc source. After adding your modules to Pantheon, add metadata, preview modules using the look and feel of the Customer Portal, associate the module with a product, publish and unpublish the module.
+
+Pantheon provides:
+
+    A GUI for managing modules and assemblies, including the ability for authors to search for specific content and other workflow features that are more user-friendly than those currently available
+    Metadata editing to optimize content for search engines
+    A module library to enable content reuse
+    A visual preview of a module so you can see how it will appear on the customer portal
+
+Pantheon does not:
+
+    Determine the appearance of content
+    Replace git as a source control system
+
+.Pantheon authoring workflow
```

Comment:
This looks like a procedure. Was that intended?

Reply:
No, but it's a series of events that takes place in a given order. Do you have a suggestion of how to better present it?

Reply:
Not in particular, no. I'm also not strictly against having this in the concept module, it just stood out to me as being a procedure. :)

Reply:
Gotcha. @benradey I see that you can either follow the current process of submitting new content to gitlab/github, or you can upload modules/assemblies directly to Pantheon by clicking *New Module* in the left nav pane. 

This is different than the current process of syncing Pantheon with the latest merged commit in the linked Gitlab repo. That raises two questions:

- Since I have to submit my content to Gitlab/Github regardless, when would I ever upload content directly to Pantheon instead of importing it from Gitlab/Github?
- Do I understand correctly the difference in process between Pantheon 1 and Pantheon 2? I.e. In Pantheon 2, I upload/import new content manually, which requires me to manually select one or more files (modules or assemblies) while in Pantheon 1 I simply sync the title to the current state of my repo?

Reply:
@stoobie Thanks for pointing that out. The "New Module" UI is a leftover prototype feature that will need to be removed before Pantheon 2 can go live. For Milestone 2, the *only* path to get content into Pantheon 2 is via git commit to a repository and then a sync in Pantheon.

To select which content is synced from git, you use the filepath patterns that are defined in pantheon2.yml, which needs to be located at the root of the repository. So that's a manual step but it only needs to be done once, not on every sync.

Does that answer your questions?

Reply:
@benradey This is news to all of us. So the CLI uploader script is also going away?

Reply:
Hi Steve, the CLI uploader is not going away. In fact it will be THE way to upload content into Pantheon.

Reply:
@stoobie The CLI script isn't going away, but the functionality that exists today (users can create publishable documents using the uploader) does not match the long term goal.

It is a design goal that all _publishable_ documents must come in through some sort of source control (one option today: git). For MS2, we only _support_ getting docs into the system through git. The CLI also currently works for that, but that is unsupported and that functionality will be disabled at some point.

Long term, we will introduce sandboxes, which are transient workspaces that house documents that _cannot_ be published, and the method to populate those will primarily be the CLI tool, among other possible methods. So the CLI tool won't go away, but it will be repurposed.

I hope that clarifies the desired/intended functionality of the system. Let me know if you still have questions.

Reply:
@carlosmunoz, @benradey Now I'm more confused. 

Ben wrote:
> For Milestone 2, the only path to get content into Pantheon 2 is via git commit to a repository and then a sync in Pantheon.

Carlos wrote:
> The CLI uploader is not going away. In fact it will be THE way to upload content into Pantheon.

Can you help me make sense of these two apparently contradictory statements?

Reply:
Not a contradiction, we were just speaking of different time frames :smiley: 
Ben gave a more detailed explanation, but the important message is that the CLI (uploader) is **not** going away.

Reply:
OK, so at the risk of beating a dead horse, which is part of my job, let me see if I understand correctly based on what the two of you have written above, keeping in mind that what I want to know is what a writer needs to know in order to do his/her job, publishing content on Pantheon 2:
1. Starting with milestone 2, which is kind of akin to an alpha release as far as writers are concerned, the only path to get content into Pantheon 2 for publishing is via git commit to a repository and then a sync in Pantheon.
1. After milestone 2, at some point in the future, in addition to publishing a document in Pantheon 2, for which I still will _only_ use git commit to a repo and then a sync in Pantheon, Another function that will be made available in Pantheon 2 is to store a document in something called a _sandbox_, which is a workspace to store one or more documents that for some reason cannot be published. The method to populate sandboxes will primarily be the CLI tool, and there might be additional methods.

Questions:
1. When do you project that sandboxes and the attendant CLI uploader be availalbe for writers to use?
1. Why do I as a writer need to have a sandbox to store docs that can't be published? If it can't be published, why wouldn't I just use a branch in git for this purpose?
1. Why would a document not be able to be published?

Reply:
After all these comments, I just removed any mention of an uploader other than Gitlab/Github.

Reply:
@stoobie I was on vacation for the last 2.5 weeks, I'm back today. You have the right idea about the path to get docs into pantheon. If you'd still like to talk about your questions, let me know and we can discuss further.